[
    {
        "id": 583396544,
        "name": "CVE-2022-36003",
        "full_name": "Live-Hack-CVE\/CVE-2022-36003",
        "owner": {
            "login": "Live-Hack-CVE",
            "id": 121191732,
            "avatar_url": "https:\/\/avatars.githubusercontent.com\/u\/121191732?v=4",
            "html_url": "https:\/\/github.com\/Live-Hack-CVE"
        },
        "html_url": "https:\/\/github.com\/Live-Hack-CVE\/CVE-2022-36003",
        "description": "TensorFlow is an open source platform for machine learning. When `RandomPoissonV2` receives large input shape and rates, it gives a `CHECK` fail that can trigger a denial of service attack. We have patched the issue in GitHub commit 552bfced6ce4809db5f3ca305f60ff80dd40c5a3. The fix will be included in TensorFlow 2.10.0 CVE project by @Sn0wAlice",
        "fork": false,
        "created_at": "2022-12-29T16:49:43Z",
        "updated_at": "2022-12-29T16:49:43Z",
        "pushed_at": "2022-12-29T16:49:45Z",
        "stargazers_count": 0,
        "watchers_count": 0,
        "has_discussions": false,
        "forks_count": 0,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [],
        "visibility": "public",
        "forks": 0,
        "watchers": 0,
        "score": 0
    }
]